datalad/core/local/run.py

```diff
@@ -127,6 +127,32 @@ class Run(Interface):
 
         % datalad run "echo my name is {name} >me"
     """
+    _examples_ = [
+        dict(text="Run an executable script and record the impact on a dataset",
+             code_py="ds.run(message='run my script', cmd='code/script.sh')",
+             code_cmd="datalad run -m 'run my script' 'code/script.sh'"),
+        dict(text="""Run a command and specify a directory as a dependency
+             for the run. The contents of the dependency will be retrieved
+             prior to running the script""",
+             code_cmd="datalad run -m 'run my script' --input 'data/\*' "
```

Comment:
The fixup commit 1df2de1f2 (BF: escape special characters in example) suggests escaping was needed for the rst conversion.  Unfortunately this isn't the right solution because it leads to `pydoc` and `--help` incorrectly showing `\*`.

Reply:
Fixed now.